lib/rules/symbol-description.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @fileoverview Rule to enforce description with the `Symbol` object
+ * @author Jarek Rencz
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const astUtils = require("../ast-utils");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "Enforce description with the `Symbol` object",
```

Comment:
Maybe this should read: "Enforce description for the `Symbol` object"? Some native-speaker correct me if I'm wrong, but with sounds strange to me in this sentence.